graspologic/embed/case.py

```diff
@@ -0,0 +1,200 @@
+import numpy as np
+from scipy.sparse.linalg import LinearOperator, eigsh
+from sklearn.preprocessing import normalize, scale
+
+from graspologic.utils import import_graph, to_laplacian, is_almost_symmetric
+from graspologic.embed.base import BaseSpectralEmbed
+
+
+class CovariateAssistedEmbed(BaseSpectralEmbed):
+    """
+    Perform Spectral Embedding on a graph with covariates, using the regularized graph Laplacian.
+
+    The Covariate-Assisted Spectral Embedding is a k-dimensional Euclidean representation
+    of a graph based on a function of its Laplacian and a vector of covariate features
+    for each node. For more information, see [1].
```

Comment:
i'd make it even more layman-y. I don't think 'function of its Laplacian and vector of covariate features' is that infomative

Reply:
8fb1ccc